keycloak4s-admin/src/main/scala/com/fullfacing/keycloak4s.admin/services/package.scala

```diff
@@ -15,6 +18,18 @@ package object services {
   type Path = ImmutableSeq[String]
   implicit def uuidToString: UUID => String = _.toString
 
+  def extractStringFromResponse(response: Either[KeycloakError, Headers]): Either[KeycloakError, String] = response.flatMap { headers =>
+    headers
+      .get("Location")
```

Comment:
Does this return `Option[String]` ? Maybe you can `flatMap` instead of `map` and use `lastOption` ?